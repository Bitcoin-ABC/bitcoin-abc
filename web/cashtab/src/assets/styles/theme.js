export const theme = {
    primary: '#00ABE7',
    brandSecondary: '#CD0BC3',
    contrast: '#fff',
    app: {
        sidebars: `url("/cashtab_bg.png")`,
        background: '#fbfbfd',
    },
    wallet: {
        background: '#fff',
        text: {
            primary: '#273498',
            secondary: '#273498',
        },
        switch: {
            activeCash: {
                shadow:
                    'inset 8px 8px 16px #0074C2, inset -8px -8px 16px #273498',
            },
            activeToken: {
                background: '#CD0BC3',
                shadow:
                    'inset 5px 5px 11px #FF21D0, inset -5px -5px 11px #CD0BC3',
            },
            inactive: {
                background: 'linear-gradient(145deg, #eeeeee, #c8c8c8)',
            },
        },
        borders: { color: '#e2e2e2' },
        shadow: 'rgba(0, 0, 0, 1)',
    },
    tokenListItem: {
        background: '#ffffff',
        color: '',
        boxShadow:
            'rgba(0, 0, 0, 0.01) 0px 0px 1px, rgba(0, 0, 0, 0.04) 0px 4px 8px,rgba(0, 0, 0, 0.04) 0px 16px 24px, rgba(0, 0, 0, 0.01) 0px 24px 32px',
        border: '#e9eaed',
        hoverBorder: '#231F20',
    },
    footer: {
        background: '#fff',
        navIconInactive: '#949494',
    },
    forms: {
        error: '#FF21D0',
        border: '#e7edf3',
        text: '#001137',
        addonBackground: '#f4f4f4',
        addonForeground: '#3e3f42',
        selectionBackground: '#fff',
    },
    icons: { outlined: '#273498' },
    modals: {
        buttons: { background: '#fff' },
    },
    settings: { delete: '#CD0BC3' },
    qr: {
        copyBorderCash: '#00ABE7',
        copyBorderToken: '#FF21D0',
        background: '#fff',
        token: '#231F20',
        shadow:
            'rgba(0, 0, 0, 0.01) 0px 0px 1px, rgba(0, 0, 0, 0.04) 0px 4px 8px, rgba(0, 0, 0, 0.04) 0px 16px 24px, rgba(0, 0, 0, 0.01) 0px 24px 32px',
    },
    buttons: {
        primary: {
            backgroundImage:
                'linear-gradient(270deg, #0074C2 0%, #273498 100%)',
            color: '#fff',
            hoverShadow: '0px 3px 10px -5px rgba(0, 0, 0, 0.75)',
        },
        secondary: {
            background: '#e9eaed',
            color: '#444',
            hoverShadow: '0px 3px 10px -5px rgba(0, 0, 0, 0.75)',
        },
    },
    collapses: {
        background: '#fbfcfd',
        border: '#eaedf3',
        color: '#3e3f42',
    },
};
