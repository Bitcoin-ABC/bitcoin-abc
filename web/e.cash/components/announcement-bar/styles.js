// Copyright (c) 2023 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.
import styled from 'styled-components';

export const AnnouncementBarCtn = styled.div`
    a,
    div {
        --barHeight: 44px;
        width: 100%;
        background-color: ${props => props.theme.colors.black};
        padding: 0px 24px;
        text-align: center;
        color: ${props => props.theme.colors.contrast};
        font-weight: 600;
        letter-spacing: 1px;
        text-transform: uppercase;
        font-size: 14px;
        height: var(--barHeight);
        display: flex;
        align-items: center;
        justify-content: center;
        transition: all 200ms ease-in-out;
        margin-top: ${props =>
            props.navBackground ? `calc(var(--barHeight) * -1)` : '0'};
    }
    a {
        :hover {
            background-color: ${props => props.theme.colors.accent};
            color: ${props => props.theme.colors.contrast};
        }
    }

    ${props => props.theme.breakpoint.medium} {
        a,
        div {
            padding: 0px 10px;
            font-size: 12px;
            --barHeight: 40px;
        }
    }

    a > span {
        display: flex;
        flex-direction: column;
        gap: 2px;
    }

    a > span span {
        line-height: 1em;
    }
`;

export const Timer = styled.span`
    color: ${props => props.theme.colors.primaryLight};
`;
