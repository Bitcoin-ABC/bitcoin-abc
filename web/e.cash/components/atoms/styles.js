// Copyright (c) 2023 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.
import styled from 'styled-components';

export const Container = styled.div`
    width: 100%;
    max-width: ${props => (props.narrow ? '900px' : '1500px')};
    margin: auto;
    padding: 0 50px;

    ${props => props.theme.breakpoint.medium} {
        padding: 0 20px;
    }
`;

export const ThemeSwitch = styled.div`
    position: fixed;
    bottom: 30px;
    width: 40px;
    height: 40px;
    z-index: 9999;
    background: rgba(255, 255, 255, 0.1);
    right: 30px;
`;

export const CenterImage = styled.div`
    position: relative;
    width: 100%;
    height: ${props => props.height};
    ${props => props.theme.filters.grayscale}

    img {
        object-fit: contain;
    }

    ${props => props.theme.breakpoint.medium} {
        height: calc(${props => props.height} / 2);
    }
`;

export const GradientSpacer = styled.div`
    height: 100px;
    width: 100%;
    background-image: linear-gradient(
        180deg,
        ${props => props.theme.colors.darkBlue},
        ${props => props.theme.colors.darkBackground}
    );
    margin-bottom: 80px;
`;

export const WarningBox = styled.p`
    padding: 20px;
    background-color: ${props => props.theme.colors.primary};
    border-radius: 10px;
    line-height: 1.5em;
`;
