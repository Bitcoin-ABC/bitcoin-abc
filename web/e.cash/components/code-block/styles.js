// Copyright (c) 2023 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.
import styled from 'styled-components';

export const Pre = styled.pre`
    background: ${props => props.theme.colors.black};
    width: 100%;
    overflow-x: scroll;
    padding: 20px;
`;
