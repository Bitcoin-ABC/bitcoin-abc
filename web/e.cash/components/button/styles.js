// Copyright (c) 2023 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.
import styled from 'styled-components';
import Link from 'next/link';
import { stealth } from '/styles/theme';

export const ButtonCtn = styled.div`
    ${props =>
        props.glow &&
        `filter: drop-shadow(0px 0px 10px ${
            props.color === 'accent'
                ? props.theme.colors.accent + '90'
                : props.color === 'white'
                ? props.theme.colors.contrast + '90'
                : props.theme.colors.primaryLight + '90'
        });`}
`;

export const ButtonMain = styled(Link)`
    display: inline-flex;
    padding: 2px;
    align-items: center;
    transition: all 300ms ease-in-out;
    color: ${props =>
        props.color === 'accent'
            ? props.theme.colors.accent
            : props.color === 'white'
            ? props.theme.colors.contrast
            : props.theme.colors.primaryLight};
    font-size: 15px;
    line-height: 1.5;
    font-weight: 400;
    letter-spacing: 1px;
    background-color: ${props =>
        props.color === 'accent'
            ? props.theme.colors.accent
            : props.color === 'white'
            ? props.theme.colors.contrast
            : props.theme.colors.primaryLight};
    ${props =>
        props.theme === stealth
            ? 'background-color: #fff !important; color: #fff !important; '
            : null};
    :hover {
        color: ${props =>
            props.color === 'accent'
                ? props.theme.colors.accent
                : props.color === 'white'
                ? props.theme.colors.contrast
                : props.theme.colors.primaryLight};
    }
`;

export const ButtonInner = styled.div`
    background-color: ${props => props.theme.colors.darkBlue};
    padding: 16px 40px;
    font-size: 15px;
    line-height: 1.5;
    font-weight: 500;
    letter-spacing: 1px;
    transition: all ease-in-out 150ms;

    &:hover {
        box-shadow: inset 0 0 13px 2px
            ${props =>
                props.color === 'accent'
                    ? props.theme.colors.accent + '90'
                    : props.color === 'white'
                    ? props.theme.colors.contrast + '90'
                    : props.theme.colors.primaryLight + '90'};
        animation: hoverbutton 150ms ease-in-out 1;
    }

    @keyframes hoverbutton {
        0% {
            background-color: ${props =>
                props.color === 'accent'
                    ? props.theme.colors.accent
                    : props.color === 'white'
                    ? props.theme.colors.contrast
                    : props.theme.colors.primaryLight} !important;
        }
        100% {
            background-color: unset;
        }
    }
    ${props => props.theme.breakpoint.medium} {
        .buttoninner {
            padding: 14px 30px;
            font-size: 14px;
        }
    }
`;
