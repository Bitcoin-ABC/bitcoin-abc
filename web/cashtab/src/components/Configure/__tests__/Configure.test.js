import React from 'react';
import renderer from 'react-test-renderer';
import Configure from '../Configure';
import { ThemeProvider } from 'styled-components';
import { theme } from '@assets/styles/theme';
let realUseContext;
let useContextMock;
beforeEach(() => {
    realUseContext = React.useContext;
    useContextMock = React.useContext = jest.fn();
});
afterEach(() => {
    React.useContext = realUseContext;
});

test('Configure without a wallet', () => {
    useContextMock.mockReturnValue({ wallet: undefined });
    const component = renderer.create(
        <ThemeProvider theme={theme}>
            <Configure />
        </ThemeProvider>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('Configure with a wallet', () => {
    useContextMock.mockReturnValue({ wallet: { mnemonic: 'test mnemonic' } });
    const component = renderer.create(
        <ThemeProvider theme={theme}>
            <Configure />
        </ThemeProvider>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
