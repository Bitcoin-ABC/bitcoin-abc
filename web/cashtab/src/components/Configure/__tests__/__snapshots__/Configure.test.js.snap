// Jest Snapshot v1, https://goo.gl/fbAQLP @generated

exports[`Configure with a wallet 1`] = `
<div
  className="sc-kEYyzF jqzglm"
>
  <h2>
    <span
      aria-label="copy"
      className="anticon anticon-copy sc-bdVaJa loPqmo"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="copy"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
        />
      </svg>
    </span>
     Backup your wallet
  </h2>
  <div
    className="ant-alert ant-alert-warning ant-alert-with-description"
    data-show={true}
    role="alert"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <span
      aria-label="exclamation-circle"
      className="anticon anticon-exclamation-circle ant-alert-icon"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="exclamation-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
        />
        <path
          d="M464 688a48 48 0 1096 0 48 48 0 10-96 0zm24-112h48c4.4 0 8-3.6 8-8V296c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8z"
        />
      </svg>
    </span>
    <div
      className="ant-alert-content"
    >
      <div
        className="ant-alert-message"
      />
      <div
        className="ant-alert-description"
      >
        Your seed phrase is the only way to restore your wallet. Write it down. Keep it safe.
      </div>
    </div>
  </div>
  <div
    className="ant-collapse ant-collapse-icon-position-left sc-EHOje iPpwBT"
    role={null}
  >
    <div
      className="ant-collapse-item"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-label="right"
          className="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        Click to reveal seed phrase
      </div>
    </div>
  </div>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  <h2>
    <span
      aria-label="wallet"
      className="anticon anticon-wallet sc-htpNat fLtATP"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="wallet"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 464H528V448h312v128zm0 264H184V184h656v200H496c-17.7 0-32 14.3-32 32v192c0 17.7 14.3 32 32 32h344v200zM580 512a40 40 0 1080 0 40 40 0 10-80 0z"
        />
      </svg>
    </span>
     Manage Wallets
  </h2>
  <button
    className="sc-VigVT RpRhe"
    onClick={[Function]}
  >
    <span
      aria-label="plus-square"
      className="anticon anticon-plus-square"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="plus-square"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
        />
        <path
          d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
        />
      </svg>
    </span>
     New Wallet
  </button>
  <button
    className="sc-jTzLTM bLkbAy"
    onClick={[Function]}
  >
    <span
      aria-label="import"
      className="anticon anticon-import"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="import"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M888.3 757.4h-53.8c-4.2 0-7.7 3.5-7.7 7.7v61.8H197.1V197.1h629.8v61.8c0 4.2 3.5 7.7 7.7 7.7h53.8c4.2 0 7.7-3.4 7.7-7.7V158.7c0-17-13.7-30.7-30.7-30.7H158.7c-17 0-30.7 13.7-30.7 30.7v706.6c0 17 13.7 30.7 30.7 30.7h706.6c17 0 30.7-13.7 30.7-30.7V765.1c0-4.3-3.5-7.7-7.7-7.7zM902 476H588v-76c0-6.7-7.8-10.5-13-6.3l-141.9 112a8 8 0 000 12.6l141.9 112c5.3 4.2 13 .4 13-6.3v-76h314c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8z"
        />
      </svg>
    </span>
     Import Wallet
  </button>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  <h2>
    <span
      aria-label="dollar"
      className="anticon anticon-dollar sc-bwzfXH iTQEyo"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="dollar"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372zm47.7-395.2l-25.4-5.9V348.6c38 5.2 61.5 29 65.5 58.2.5 4 3.9 6.9 7.9 6.9h44.9c4.7 0 8.4-4.1 8-8.8-6.1-62.3-57.4-102.3-125.9-109.2V263c0-4.4-3.6-8-8-8h-28.1c-4.4 0-8 3.6-8 8v33c-70.8 6.9-126.2 46-126.2 119 0 67.6 49.8 100.2 102.1 112.7l24.7 6.3v142.7c-44.2-5.9-69-29.5-74.1-61.3-.6-3.8-4-6.6-7.9-6.6H363c-4.7 0-8.4 4-8 8.7 4.5 55 46.2 105.6 135.2 112.1V761c0 4.4 3.6 8 8 8h28.4c4.4 0 8-3.6 8-8.1l-.2-31.7c78.3-6.9 134.3-48.8 134.3-124-.1-69.4-44.2-100.4-109-116.4zm-68.6-16.2c-5.6-1.6-10.3-3.1-15-5-33.8-12.2-49.5-31.9-49.5-57.3 0-36.3 27.5-57 64.5-61.7v124zM534.3 677V543.3c3.1.9 5.9 1.6 8.8 2.2 47.3 14.4 63.2 34.4 63.2 65.1 0 39.1-29.4 62.6-72 66.4z"
        />
      </svg>
    </span>
     Fiat Currency
  </h2>
  <div
    className="sc-iwsKbI dmHKdG"
  >
    <div
      className="ant-select select-after ant-select-single ant-select-show-arrow"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <div
        className="ant-select-selector"
        onClick={[Function]}
        onMouseDown={[Function]}
      >
        <span
          className="ant-select-selection-search"
        >
          <input
            aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
            aria-autocomplete="list"
            aria-controls="rc_select_TEST_OR_SSR_list"
            aria-haspopup="listbox"
            aria-owns="rc_select_TEST_OR_SSR_list"
            autoComplete="off"
            className="ant-select-selection-search-input"
            id="rc_select_TEST_OR_SSR"
            onChange={[Function]}
            onCompositionEnd={[Function]}
            onCompositionStart={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            onPaste={[Function]}
            readOnly={true}
            role="combobox"
            style={
              Object {
                "opacity": 0,
              }
            }
            type="search"
            unselectable="on"
            value=""
          />
        </span>
        <span
          className="ant-select-selection-item"
          title="US Dollar ($)"
        >
          US Dollar ($)
        </span>
      </div>
      <span
        aria-hidden={true}
        className="ant-select-arrow"
        onMouseDown={[Function]}
        style={
          Object {
            "WebkitUserSelect": "none",
            "userSelect": "none",
          }
        }
        unselectable="on"
      >
        <span
          aria-label="down"
          className="anticon anticon-down ant-select-suffix"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="down"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  [
  <a
    className="sc-dxgOiQ jbuVAx"
    href="https://docs.cashtabapp.com/docs/"
    rel="noreferrer"
    target="_blank"
    type="link"
  >
    Documentation
  </a>
  ]
</div>
`;

exports[`Configure without a wallet 1`] = `
<div
  className="sc-kEYyzF jqzglm"
>
  <h2>
    <span
      aria-label="copy"
      className="anticon anticon-copy sc-bdVaJa loPqmo"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="copy"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
        />
      </svg>
    </span>
     Backup your wallet
  </h2>
  <div
    className="ant-alert ant-alert-warning ant-alert-with-description"
    data-show={true}
    role="alert"
    style={
      Object {
        "marginBottom": "12px",
      }
    }
  >
    <span
      aria-label="exclamation-circle"
      className="anticon anticon-exclamation-circle ant-alert-icon"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="exclamation-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
        />
        <path
          d="M464 688a48 48 0 1096 0 48 48 0 10-96 0zm24-112h48c4.4 0 8-3.6 8-8V296c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8z"
        />
      </svg>
    </span>
    <div
      className="ant-alert-content"
    >
      <div
        className="ant-alert-message"
      />
      <div
        className="ant-alert-description"
      >
        Your seed phrase is the only way to restore your wallet. Write it down. Keep it safe.
      </div>
    </div>
  </div>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  <h2>
    <span
      aria-label="wallet"
      className="anticon anticon-wallet sc-htpNat fLtATP"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="wallet"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 464H528V448h312v128zm0 264H184V184h656v200H496c-17.7 0-32 14.3-32 32v192c0 17.7 14.3 32 32 32h344v200zM580 512a40 40 0 1080 0 40 40 0 10-80 0z"
        />
      </svg>
    </span>
     Manage Wallets
  </h2>
  <button
    className="sc-VigVT RpRhe"
    onClick={[Function]}
  >
    <span
      aria-label="plus-square"
      className="anticon anticon-plus-square"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="plus-square"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M328 544h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
        />
        <path
          d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
        />
      </svg>
    </span>
     New Wallet
  </button>
  <button
    className="sc-jTzLTM bLkbAy"
    onClick={[Function]}
  >
    <span
      aria-label="import"
      className="anticon anticon-import"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="import"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M888.3 757.4h-53.8c-4.2 0-7.7 3.5-7.7 7.7v61.8H197.1V197.1h629.8v61.8c0 4.2 3.5 7.7 7.7 7.7h53.8c4.2 0 7.7-3.4 7.7-7.7V158.7c0-17-13.7-30.7-30.7-30.7H158.7c-17 0-30.7 13.7-30.7 30.7v706.6c0 17 13.7 30.7 30.7 30.7h706.6c17 0 30.7-13.7 30.7-30.7V765.1c0-4.3-3.5-7.7-7.7-7.7zM902 476H588v-76c0-6.7-7.8-10.5-13-6.3l-141.9 112a8 8 0 000 12.6l141.9 112c5.3 4.2 13 .4 13-6.3v-76h314c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8z"
        />
      </svg>
    </span>
     Import Wallet
  </button>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  <h2>
    <span
      aria-label="dollar"
      className="anticon anticon-dollar sc-bwzfXH iTQEyo"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="dollar"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372zm47.7-395.2l-25.4-5.9V348.6c38 5.2 61.5 29 65.5 58.2.5 4 3.9 6.9 7.9 6.9h44.9c4.7 0 8.4-4.1 8-8.8-6.1-62.3-57.4-102.3-125.9-109.2V263c0-4.4-3.6-8-8-8h-28.1c-4.4 0-8 3.6-8 8v33c-70.8 6.9-126.2 46-126.2 119 0 67.6 49.8 100.2 102.1 112.7l24.7 6.3v142.7c-44.2-5.9-69-29.5-74.1-61.3-.6-3.8-4-6.6-7.9-6.6H363c-4.7 0-8.4 4-8 8.7 4.5 55 46.2 105.6 135.2 112.1V761c0 4.4 3.6 8 8 8h28.4c4.4 0 8-3.6 8-8.1l-.2-31.7c78.3-6.9 134.3-48.8 134.3-124-.1-69.4-44.2-100.4-109-116.4zm-68.6-16.2c-5.6-1.6-10.3-3.1-15-5-33.8-12.2-49.5-31.9-49.5-57.3 0-36.3 27.5-57 64.5-61.7v124zM534.3 677V543.3c3.1.9 5.9 1.6 8.8 2.2 47.3 14.4 63.2 34.4 63.2 65.1 0 39.1-29.4 62.6-72 66.4z"
        />
      </svg>
    </span>
     Fiat Currency
  </h2>
  <div
    className="sc-iwsKbI dmHKdG"
  >
    <div
      className="ant-select select-after ant-select-single ant-select-show-arrow"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <div
        className="ant-select-selector"
        onClick={[Function]}
        onMouseDown={[Function]}
      >
        <span
          className="ant-select-selection-search"
        >
          <input
            aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
            aria-autocomplete="list"
            aria-controls="rc_select_TEST_OR_SSR_list"
            aria-haspopup="listbox"
            aria-owns="rc_select_TEST_OR_SSR_list"
            autoComplete="off"
            className="ant-select-selection-search-input"
            id="rc_select_TEST_OR_SSR"
            onChange={[Function]}
            onCompositionEnd={[Function]}
            onCompositionStart={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            onPaste={[Function]}
            readOnly={true}
            role="combobox"
            style={
              Object {
                "opacity": 0,
              }
            }
            type="search"
            unselectable="on"
            value=""
          />
        </span>
        <span
          className="ant-select-selection-item"
          title="US Dollar ($)"
        >
          US Dollar ($)
        </span>
      </div>
      <span
        aria-hidden={true}
        className="ant-select-arrow"
        onMouseDown={[Function]}
        style={
          Object {
            "WebkitUserSelect": "none",
            "userSelect": "none",
          }
        }
        unselectable="on"
      >
        <span
          aria-label="down"
          className="anticon anticon-down ant-select-suffix"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="down"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
  <div
    className="sc-kkGfuU lbBTMM"
  />
  [
  <a
    className="sc-dxgOiQ jbuVAx"
    href="https://docs.cashtabapp.com/docs/"
    rel="noreferrer"
    target="_blank"
    type="link"
  >
    Documentation
  </a>
  ]
</div>
`;
